lib/curl_libproxy.c

```diff
@@ -0,0 +1,69 @@
+/***************************************************************************
+ *                                  _   _ ____  _
+ *  Project                     ___| | | |  _ \| |
+ *                             / __| | | | |_) | |
+ *                            | (__| |_| |  _ <| |___
+ *                             \___|\___/|_| \_\_____|
+ *
+ * Copyright (C) 2022-2023, Jan-Michael Brummer, <jan-michael.brummer1@volkswagen.de>.
```

Comment:
We don't use year ranges in our copyright lines anymore.

Reply:
Used checksrc and fixed the nitpicks.